### ISSUE TITLE
no-extra-parens false positive

### ISSUE THREAD

**Issue by jpzwarte** [eslint/eslint#6219]
```
ratio = (img.naturalWidth / img.naturalHeight) / (rect.width / rect.height);
```

The above triggers a false positive for the no-extra-parens rule: if you remove the parens, the value is different.

**What version of ESLint are you using?**
2.10.2

**What parser (default, Babel-ESLint, etc.) are you using?**
ecmaVersion: 6

**Please show your full configuration:**
https://gist.github.com/jpzwarte/46351dfd29063ceaf3f2447916112a22

**What actually happened? Please include the actual, raw output from ESLint.**
header.js
17:23  error  Gratuitous parentheses around expression  no-extra-parens

**Comment by mysticatea** [comment]
Thank you for this issue.

> 17:23 error Gratuitous parentheses around expression no-extra-parens

Where is `17:23`?

**Comment by jpzwarte** [comment]
That's the line i pasted at the top of the issue.

**Comment by mysticatea** [comment]
I'd like to see the exact position. Is it the first open paren?

**Comment by mysticatea** [comment]
Because the first parentheses in your code is surely extra parentheses.
You can remove it safely.

**Comment by jpzwarte** [comment]
I made a simpler example:

```
3:9  error  Gratuitous parentheses around expression         no-extra-parens
```

```
var a = (1 / 2) / (1 / 2);
```

So 9 is the first opening parens.

**Comment by jpzwarte** [comment]
Ah, you'r absolutely right. First parens is superfluous, but the second isn't. Thanks.

**Comment by platinumazure** [comment]
@jpzwarte Also remember that you can always use inline configuration comments if you honestly believe that the code reads better with the rule violation:

```js
var result = (1 / 2) / (1 / 2); // eslint-disable-line no-extra-semi
```
